Personal page for uchechukwu

Uchechukwu-Ekezie opened the pull request:
This PR adds my personal builder page for Batch 22 of BuidlGuidl. The page features:

Two-card layout with profile information and "About Me" section
Gradient title with custom styling using accent, info, and success colors
Avatar with "CD" initials in a gradient circle
Social links with GitHub and Twitter buttons (with SVG icons)
Responsive design that adapts to mobile and desktop views
Enhanced styling with shadows, borders, and hover effects
TypeScript with proper NextPage typing
The page is located at /builders/0x3Be7fbBDbC73Fc4731D60EF09c4BA1A94DC58E41 and includes:

Welcome section with gradient title
Profile card with avatar, address, bio, and social links
About Me card with sections for Learning, Interests, and Goals

<img width="1379" height="872" alt="image" src="https://github.com/user-attachments/assets/e0aff923-c1b2-4289-a945-5e90860efc18" />

Closes https://github.com/BuidlGuidl/batch22.buidlguidl.com/issues/10

Your ENS/address:
0xa91D5A0a64ED5eeF11c4359C4631279695A338ef

melanke commented on packages/nextjs/contracts/deployedContracts.ts:
This change is also unrelated to your personal page, please remove it from the PR.

Uchechukwu-Ekezie commented:
working on it sir

am sorry for replying late

Uchechukwu-Ekezie commented:
This PR adds my personal builder page to the Batch 22 showcase.

## Changes Made
- Added personal builder page at `0xa91D5A0a64ED5eeF11c4359C4631279695A338ef`
- Removed "use client" directive for server-side rendering
- Organized icons into separate component files
- Fixed component props and TypeScript types

## Additional Information
- [x] I have read the contributing docs (if this is your first contribution)
- [x] This is not a duplicate of any existing pull request
